gap/examples.gi

```diff
@@ -412,5 +412,6 @@ InstallMethod(KingsGraph, "for a function, positive integer, positive integer",
 [IsFunction, IsPosInt, IsPosInt],
 KingsGraphCons);
 
-InstallMethod(KingsGraph, "for positive integer, positive integer", [IsPosInt, IsPosInt],
+InstallMethod(KingsGraph, "for positive integer, positive integer", 
```

Comment:
There is no exact requirement for such strings, only some conventions,  but GAP library would rather use "for two positive integers". Unless your use is consistent for Digraphs, I'd suggest to do like GAP.

Reply:
Oh ok, thanks for the tip! I will change it to "for two positive integers"